src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,67 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Using `eslint-disable` is a bag practice. Try to avoid it here and in test tasks. Find the way to solve it without `eslint-disable`.

Try to add `aria-label` attribute to buttons all over the project to avoid using `eslint-disable`.
More information  [here]( https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/control-has-associated-label.md)